src/code.sh

```
@@ -109,50 +164,48 @@ main() {
 
     if [[ "$run_CollectRnaSeqMetrics" == true ]]; then
         # Create refFlat file if not provided by user
         if [ -z "$ref_annot_refflat" ]; then
             echo "No refFlat file provided - creating GTF from refFlat file in CTAT bundle"
-            lib_dir=$(echo $fasta_index_name | cut -d "." -f 1,2)
-            ref_annot_gtf="/home/dnanexus/${lib_dir}/ctat_genome_lib_build_dir/ref_annot.gtf"
-            $java -jar /GtftoRefflat-assembly-0.1.jar \
-                -g "$ref_annot_gtf" \
-                -r "${lib_dir}_ref_annot.refflat"
-            ref_flat="${lib_dir}_ref_annot.refflat"
+            LIB_DIR=$(echo $fasta_index_name | cut -d "." -f 1,2)
```

Shellcheck (0.10.0)

[warning] fasta_index_name is referenced but not assigned.

(SC2154)